active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,146 @@
+Breadcrumb control
+===
+
+# Background
+
+There’s currently no consistent and Fluent way to address the common UX pattern of a breadcrumb.
+This control is needed for situations where the user's navigation trail (in a file system or menu system) needs
+to be persistently visible.
+
+A Breadcrumb control is movitated by its use in many app scenarios and supporting developers in migrating from WPF.
+This is not the 'history' of the user's navigation, but the direct path of pages or folders from
+the root node to their current position. 
+
+## V1/V2 Breadcrumb
+
+The work for Breadcrumb has been scoped to V1 and V2 stages of the API.
+In V1 Breadcrumb no chevrons are interactable to show children.
+V2 Breadcrumb will add functionality to have flyouts from chevrons for every node,
+to view children of that node. This spec focuses on V1 Breadcrumb,
+but keeping in mind that V1 is positioned in a way to have V2 features smoothly added later. 
+
+# API Pages
+
+## Breadcrumb
+
+Represents a control that shows a list of items, typically the navigation trail to the current location.
+
+
+### Examples
+
+The following example code creates a simple Breadcrumb control with three nodes:
+
+![Breadcrumb default with 3 nodes](images/Breadcrumb_default.PNG)
+
+XAML
+```XAML
+<Breadcrumb x:Name="Breadcrumb1" /> 
+```
+
+C#
+```cs
+Breadcrumb1.ItemsSource = new string[]{ "Node", "Node" , "Node"}
+```
+
+When there are too many items to display, a Breadcrumb "crumbles" and shows only the 
+last items.
+
+![Breadcrumb crumbling with 3 visible nodes](images/Breadcrumb_crumbling.PNG)
```

Comment:
This says 3 but looks like 2. Or does crumble not mean what I think it means?

Reply:
I'll update the description here, this was a copy-paste mistake.